src/borg/archive.py

```diff
@@ -1115,6 +1115,10 @@ def _open_rb(path):
                 raise
             # Was this EPERM due to the O_NOATIME flag? Try again without it:
             return os.open(path, flags_normal)
+        except OSError as exc:
+            # O_NOATIME causes EROFS when accessing a volume shadow copy in WSL1
+            if exc.errno == errno.EROFS:
+                return os.open(path, flags_normal)
```

Comment:
Hmm, this matches the strange behaviour you have seen on WSL1, but I think:
- this is a strange bug in WSL1
- this should not be the standard reaction of borg if it encounters a EROFS in general

So, please extend BORG_WORKAROUNDS and only implement this behaviour if specifically requested by a specific value in the workarounds list.